### PR TITLE
[indexer] Removed indexer dependency on search.

### PR DESCRIPTION
**Pull request by ygorshenin**
@milchakov PTAL

### CODE REVIEW

**Comment by ygorshenin** [comment]
DUDE JTTP

File: indexer/indexer_tests_support/helpers.cpp [hunk 1]
```diff
@@ -2,16 +2,14 @@
 
 #include "editor/editor_storage.hpp"
 
-#include "std/unique_ptr.hpp"
```
Comment (milchakov) [inline indexer/indexer_tests_support/helpers.cpp]: хм, а почему удалил?

Comment (milchakov) [inline indexer/indexer_tests_support/helpers.cpp]: а, понял, в хедер перенес

File: indexer/indexer_tests_support/helpers.hpp [hunk 2]
```diff
@@ -3,6 +3,8 @@
 #include "indexer/editable_map_object.hpp"
 #include "indexer/osm_editor.hpp"
 
+#include "std/unique_ptr.hpp"
+
 #include "base/assert.hpp"
```
Comment (milchakov) [inline indexer/indexer_tests_support/helpers.hpp]: отсортируй пожалуйста

File: search/editor_delegate.cpp [hunk 3]
```diff
@@ -0,0 +1,44 @@
+#include "search/editor_delegate.hpp"
+
```
Comment (milchakov) [inline search/editor_delegate.cpp]: лишняя строка

Comment (ygorshenin) [inline search/editor_delegate.cpp]: Not sure. The first line for cpp implementation for header should (from my point of view) be an inclusion of the header, separated from other includes by blank line.

File: indexer/indexer_tests_support/helpers.hpp (hunk 2 above)
Comment (ygorshenin) [inline indexer/indexer_tests_support/helpers.hpp]: Done

**Comment by ygorshenin** [comment]
PTAL

**Pull request merged**